manifests/blacklist.pp

```diff
@@ -1,10 +1,13 @@
 # ex: si ts=4 sw=4 et
 
 define shorewall::blacklist (
-    $type         = 'ipv4',
+    $action        = 'DROP',
+    $type          = 'ipv4',
+    $source        = 'all',
     $address       = '',
+    $dest          = '$FW',
     $proto         = '',
-    $port          = '',
+    $port          = [],
```

Comment:
I'm not sure what you're doing exactly, but it isn't updating the PR. I see empty merge commits.